### PR TITLE
Replaced gogits go-gogs-client with go-sdk

### LINKED ISSUE #93: Replace gogits go-gogs-client with go-sdk
We want to replace `gogits/go-gogs-client` with `go-gitea/go-sdk`, below you can see a greped list of values. Please keep in mind that we also need to update our client repository.

```
# grep -R go-gogs-client *
README.md:5. If you're interested in using APIs, we have experimental support with [documentation](https://github.com/gogits/go-gogs-client/wiki).
models/action.go:	api "github.com/gogits/go-gogs-client"
models/issue.go:	api "github.com/gogits/go-gogs-client"
models/issue_comment.go:	api "github.com/gogits/go-gogs-client"
models/issue_label.go:	api "github.com/gogits/go-gogs-client"
models/pull.go:	api "github.com/gogits/go-gogs-client"
models/repo.go:	api "github.com/gogits/go-gogs-client"
models/user.go:	api "github.com/gogits/go-gogs-client"
models/webhook.go:	api "github.com/gogits/go-gogs-client"
models/webhook_slack.go:	api "github.com/gogits/go-gogs-client"
modules/base/base.go:const DOC_URL = "https://github.com/gogits/go-gogs-client/wiki"
routers/api/v1/admin/org.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/admin/org.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Organizations#create-a-new-organization
routers/api/v1/admin/org_team.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/admin/repo.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/admin/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Repositories#create-a-new-repository
routers/api/v1/admin/user.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/admin/user.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Users#create-a-new-user
routers/api/v1/admin/user.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Users#edit-an-existing-user
routers/api/v1/admin/user.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Users#delete-a-user
routers/api/v1/admin/user.go:// https://github.com/gogits/go-gogs-client/wiki/Administration-Users#create-a-public-key-for-user
routers/api/v1/api.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/convert/convert.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/misc/markdown.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/misc/markdown.go:// https://github.com/gogits/go-gogs-client/wiki/Miscellaneous#render-an-arbitrary-markdown-document
routers/api/v1/misc/markdown.go:// https://github.com/gogits/go-gogs-client/wiki/Miscellaneous#render-a-markdown-document-in-raw-mode
routers/api/v1/org/org.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/org/org.go:// https://github.com/gogits/go-gogs-client/wiki/Organizations#list-your-organizations
routers/api/v1/org/org.go:// https://github.com/gogits/go-gogs-client/wiki/Organizations#list-user-organizations
routers/api/v1/org/org.go:// https://github.com/gogits/go-gogs-client/wiki/Organizations#get-an-organization
routers/api/v1/org/org.go:// https://github.com/gogits/go-gogs-client/wiki/Organizations#edit-an-organization
routers/api/v1/org/team.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/branch.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/branch.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#get-branch
routers/api/v1/repo/branch.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#list-branches
routers/api/v1/repo/collaborators.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/file.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Contents#download-raw-content
routers/api/v1/repo/file.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Contents#download-archive
routers/api/v1/repo/hook.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/hook.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#list-hooks
routers/api/v1/repo/hook.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#create-a-hook
routers/api/v1/repo/hook.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#edit-a-hook
routers/api/v1/repo/issue.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/issue_comment.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/issue_label.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/key.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/key.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Deploy-Keys#list-deploy-keys
routers/api/v1/repo/key.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Deploy-Keys#get-a-deploy-key
routers/api/v1/repo/key.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Deploy-Keys#add-a-new-deploy-key
routers/api/v1/repo/key.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories-Deploy-Keys#remove-a-deploy-key
routers/api/v1/repo/label.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/milestone.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/repo.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#search-repositories
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#list-your-repositories
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#create
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#migrate
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#get
routers/api/v1/repo/repo.go:// https://github.com/gogits/go-gogs-client/wiki/Repositories#delete
routers/api/v1/user/app.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/user/app.go:// https://github.com/gogits/go-gogs-client/wiki/Users#list-access-tokens-for-a-user
routers/api/v1/user/app.go:// https://github.com/gogits/go-gogs-client/wiki/Users#create-a-access-token
routers/api/v1/user/email.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/user/email.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Emails#list-email-addresses-for-a-user
routers/api/v1/user/email.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Emails#add-email-addresses
routers/api/v1/user/email.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Emails#delete-email-addresses
routers/api/v1/user/follower.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#list-followers-of-a-user
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#list-users-followed-by-another-user
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#check-if-you-are-following-a-user
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#check-if-one-user-follows-another
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#follow-a-user
routers/api/v1/user/follower.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Followers#unfollow-a-user
routers/api/v1/user/key.go:	api "github.com/gogits/go-gogs-client"
routers/api/v1/user/key.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Public-Keys#list-your-public-keys
routers/api/v1/user/key.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Public-Keys#list-public-keys-for-a-user
routers/api/v1/user/key.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Public-Keys#get-a-single-public-key
routers/api/v1/user/key.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Public-Keys#create-a-public-key
routers/api/v1/user/key.go:// https://github.com/gogits/go-gogs-client/wiki/Users-Public-Keys#delete-a-public-key
routers/api/v1/user/user.go:	api "github.com/gogits/go-gogs-client"
routers/repo/webhook.go:	api "github.com/gogits/go-gogs-client"
routers/repo/webhook.go:	ctx.Data["Description"] = ctx.Tr("repo.settings.hooks_desc", "https://github.com/gogits/go-gogs-client/wiki/Repositories-Webhooks")
templates/repo/create.tmpl:						<label>{{.i18n.Tr "repo.readme"}} <a target="_blank" href="https://github.com/gogits/go-gogs-client/wiki/Repositories#litte-notes-on-readme-template"><span class="octicon octicon-question"></span></a></label>
vendor/vendor.json:			"path": "github.com/gogits/go-gogs-client",
```

### PR DESCRIPTION
**Pull request by tboerger**
Fixes #93

### CODE REVIEW

**Comment by strk** [comment]
Why not keep the `api` alias ? would make the patch much smaller

**Comment by tboerger** [comment]
It will change again with go-gitea/go-sdk#5

**Review by bkcsoft — COMMENTED** [review]
2 questions :)

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 2. Due to testing purpose, data of [try.gogs.io](https://try.gogs.io) was reset in **Jan 28, 2015** and will reset multiple times after. Please do **NOT** put your important data on the site.
 3. The demo site [try.gogs.io](https://try.gogs.io) is running under `develop` branch.
 4. If you think there are vulnerabilities in the project, please talk privately to **u@gogs.io**. Thanks!
```
[the remark this answers is not in the document]
Comment (tboerger) [reply README.md:25]: That's already part of a different pr

**Comment by tboerger** [comment]
I have updated the PR to only change the import path and to keep the `api` alias.

**Comment by strk** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**